examples/clock/README.md

```diff
@@ -0,0 +1,7 @@
+# Clock example
+
+This is a simple example showcasing the Gloo timers.
+
+First, install wasm-pack if needed by running `cargo install wasm-pack`.
```

Comment:
Could this be changed to link to the [the wasm-pack install page](https://rustwasm.github.io/wasm-pack/installer/)?

Reply:
done, but still some CI failures.

Reply:
ping @Pauan